builtin/aws/ecs/platform.go

```diff
@@ -3013,6 +3085,38 @@ deploy {
 		}),
 	)
 
+	doc.SetField("health_check",
+		"Health check settings for the app.",
+		docs.SubFields(func(doc *docs.SubFieldDoc) {
+			doc.SetField("protocol",
+				"The protocol for the health check to use.")
```

Comment:
Not sure if our docs have a special field for this, but it would be nice to say what the valid protocols are!